lib/ActivityStream.jsm

```diff
@@ -222,10 +219,11 @@ const PREFS_CONFIG = new Map([
     title: "Configuration for the new pocket new tab",
     getValue: ({geo, locale}) => {
       const locales = ({
-        "US": ["en-CA", "en-GB", "en-US", "en-ZA"],
-        "CA": ["en-CA", "en-GB", "en-US", "en-ZA"],
+        "US": ["en-US"],
       })[geo];
-      const isEnabled = IS_NIGHTLY_OR_UNBRANDED_BUILD && locales && locales.includes(locale);
+
+      const isEnabled = locales && locales.includes(locale);
```

Comment:
Small nit, I wonder if we keep some of the existing logic, we could keep this on for nightly in en-CA.

Something like?

```
const isEnabled = (locale === "en-US") || (locales && locales.includes(locale));
```

Reply:
I don't think that's the intention though. See [Slack thread](https://mozilla.slack.com/archives/CEZEF1AHX/p1561660281201100)

Reply:
Nevermind...that is what's wanted.